torchvision/transforms.py

```diff
@@ -260,7 +344,7 @@ def __call__(self, img):
         Returns:
             PIL.Image: Padded image.
         """
-        return ImageOps.expand(img, border=self.padding, fill=self.fill)
+        return pad(img, self.padding, self.fill)
 
 
 class Lambda(object):
```

Comment:
If we are starting to move towards a functional interface, perhaps we should just remove the `Lambda` transform altogether (maybe for later and for another release)?

Reply:
I think it is still worth it to keep `Lambda` transform around for the moment, but in the future we might deprecate it.